src/tools/sfdk/command.cpp

```diff
@@ -1660,12 +1660,24 @@ Worker::ExitStatus BuiltinWorker::runTools(const QStringList &arguments_, int *e
     }
 
     if (arguments.first() == "remove") {
-        if (!P::checkPositionalArgumentsCount(arguments, 2, 2))
+        QCommandLineParser parser;
+        QCommandLineOption snapshotsOfOption(QStringList{"snapshots-of"});
+        parser.addOption(snapshotsOfOption);
+        parser.addPositionalArgument("name", QString(), "[name]");
+
+        if (!parser.parse(arguments)) {
+             qerr() << parser.errorText() << endl;
+             return BadUsage;
+        }
+
+        QStringList positionalArguments = parser.positionalArguments();
+        if (positionalArguments.count() != 1)
```

Comment:
Unless I missed something, `P::checkPositionalArguments` should be used as it also prints an appropriate message in case of error. I would also omit the variable for consistency - `QCommandLineParser::positionalArguments` is likely an inline function anyway.

```suggestion
        if (!P::checkPositionalArgumentsCount(parser.positionalArguments(), 1, 1))
```